app/javascript/components/HelloWorld.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from "react"
+import PropTypes from "prop-types"
+import { useSelector, shallowEqual, useDispatch } from 'react-redux';
+import { fetchGreetingApi } from "../redux/greetings/greeting";
+
+const HelloWorld = () => {
```

Comment:
- [OPTIONAL] This component is responsible to handle greetings. Because of this reason, I will suggest you rename it from `HelloWorld` to `Greeting`